src/Rector/Tca/AbstractTcaRector.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+namespace Ssch\TYPO3Rector\Rector\Tca;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr;
+use PhpParser\Node\Expr\Array_;
+use PhpParser\Node\Expr\ArrayItem;
+use PhpParser\Node\Scalar\String_;
+use Rector\Core\Rector\AbstractRector;
+use Ssch\TYPO3Rector\Helper\TcaHelperTrait;
+
+/**
+ * @changelog
```

Comment:
You can exclude some classes to having a @changelog. Add it here: https://github.com/sabbelasichon/typo3-rector/blob/main/utils/phpstan/src/Rules/AddChangelogDocBlockForRectorClass.php#L32